composer.json

```diff
@@ -1,23 +1,25 @@
 {
     "name": "lcobucci/jwt",
-    "description": "A simple library to work with JSON Web Token and JSON Web Signature",
     "type": "library",
-    "authors": [
-        {
-            "name": "Luís Cobucci",
-            "email": "lcobucci@gmail.com",
-            "role": "Developer"
-        }
-    ],
+    "description": "A simple library to work with JSON Web Token and JSON Web Signature",
     "keywords": [
         "JWT",
         "JWS"
     ],
     "license": [
         "BSD-3-Clause"
     ],
+    "authors": [
+        {
+            "name": "Luís Cobucci",
+            "email": "lcobucci@gmail.com",
+            "role": "Developer"
+        }
+    ],
     "require": {
         "php": "^7.4 || ^8.0",
+        "ext-hash": "*",
+        "ext-json": "*",
```

Comment:
All other changes are just formal, but these two are substantial and very needed